system/core-workflows/how-do-they-work.rst

```diff
@@ -189,15 +190,20 @@ unset readonly |ui|
 Stop after match
 ----------------
 
-Stop evaluation of other, following workflows that would match otherwise.
+Here you can decide if other workflows are executed after the current one.
+
+If set to ``no`` (default), no further worflows will be executed after the
+current one.
 
-Default: ``no``
+If set to ``yes``, further workflows will be executed if they match the
+condition. In this case, it is possible that your actions from the current
+workflow can be overwritten by another workflow.
```

Comment:
this looks odd. I think it is the other way around :D

yes will stop
no will continue 


```
If set to ``no`` (default), further workflows will be executed if they match the
condition. In this case, it is possible that your actions from the current
workflow can be overwritten by another workflow.

If set to ``yes``, no further worflows will be executed after the
current one.
```